recipes/cairo/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "1.12.18" %}
+
+package:
+  name: cairo
+  version: {{ version }}
+
+source:
+  fn: cairo-{{ version }}.tar.xz
+  url: http://cairographics.org/releases/cairo-{{ version }}.tar.xz
+  md5: 8e4ff32b82c3b39387eb6f5c59ef848e
+
+build:
+  number: 0
+  skip: true             # [win]
+  features:
+    - vc9                # [win and py27]
+    - vc10               # [win and py34]
+    - vc14               # [win and py35]
+
+requirements:
+  build:
+    - python                  # [win]
+    - pkg-config              # [unix]
+    - xz                      # [unix]
+    - freetype 2.5.*          # [unix]
+    - fontconfig 2.11.*       # [unix]
+    - pixman                  # [unix]
+    - libpng 1.6*
+    - zlib 1.2*
+  run:
+    - freetype                # [unix]
+    - fontconfig 2.11.*       # [unix]
+    - libpng 1.6*
+    - pixman                  # [unix]
+    - zlib 1.2*
+
+test:
+  commands:
+    # Check commands.
+    - cairo-trace --help
+
+    # Verify libraries.
+    {% set cairo_libs = [
+            "libcairo",
+            "libcairo-script-interpreter",
+    ] %}
+    {% for each_cairo_lib in cairo_libs %}
+    - test -f $PREFIX/lib/{{ each_cairo_lib }}.a                                                 # [unix]
+    - test -f $PREFIX/lib/{{ each_cairo_lib }}.dylib                                             # [osx]
+    - test -f $PREFIX/lib/{{ each_cairo_lib }}.so                                                # [linux]
+    {% endfor %}
+
+about:
+  home: http://cairographics.org/
+  license: LGPL 2.1 or MPL 1.1
+  summary: Cairo is a 2D graphics library with support for multiple output devices.
+
+extra:
+  recipe-maintainers:
+    - ccordoba12
```

Comment:
@ccordoba12, I have added you to `cairo` as you seem to be the most interested in this based on the history in `conda-recipes`. Also, I think you have a better idea of what is going on with Windows here. This should give you the access necessary to fix it as needed.